instrumentation/que/lib/opentelemetry/instrumentation/que/patches/que_job.rb

```diff
@@ -67,6 +60,32 @@ def enqueue(*args, job_options: {}, **arg_opts)
             def gem_version
               @gem_version ||= Gem.loaded_specs['que'].version
             end
+
+            if Gem.loaded_specs['que'].version >= Gem::Version.new('2.1.0')
+              def bulk_enqueue(**_kwargs, &block)
+                tracer = Que::Instrumentation.instance.tracer
+                otel_config = Que::Instrumentation.instance.config
+
+                tracer.in_span('publish', kind: :producer) do |span|
+                  super do
+                    yield
+
+                    job_attrs = Thread.current[:que_jobs_to_bulk_insert][:jobs_attrs]
+
+                    unless job_attrs.empty?
+                      span.name = "#{job_attrs.first[:job_class]} publish"
+                      span.add_attributes(QueJob.job_attributes(job_attrs.first))
```

Comment:
Does it add `messaging.message_id` to the attributes? It shouldn't in case of a bulk enqueue. The other properties seem correct because they should be the same according to the bulk enqueue [limitations](https://github.com/que-rb/que/blob/master/docs/README.md#enqueueing-jobs-in-bulk).